### PR TITLE
Modernize

### PR DESCRIPTION
**Pull request by seguri**

### CODE REVIEW

**Comment by brock** [comment]
Please add yourself to the list of DJ's in the "Featuring" section at the top of the Readme if you'd like. And thanks @seguri !!

**Review by brock — COMMENTED** [review]
We should probably add these two lines to the top of the readme as well, since we call out what is getting deleted there.

File: node-reinstall [hunk 1]
```diff
@@ -193,17 +193,14 @@ rm -rf $HOME/{local,lib,include,node*,npm,.npm*,.node-gyp}
 sudo rm -rf $PREFIX/lib/node*
 sudo rm -rf $PREFIX/include/node*
 sudo rm -rf $PREFIX/bin/{node,npm}
+sudo rm -rf $PREFIX/share/doc/node
```
Comment (brock) [inline node-reinstall:196]: These directories getting deleted should be added to the README in the block showing what gets deleted

Comment (seguri) [reply node-reinstall:196]: Done

**Comment by brock** [comment]
Thanks again @seguri

**Pull request merged**